nodejs/webrtc-hello-world/src/server.ts

```diff
@@ -3,7 +3,7 @@ import dotenv from "dotenv";
 import express from "express";
 import bodyParser from "body-parser";
 import axios from "axios";
-import jwt_decode from "jwt-decode";
+const bandwidthWebRTC = require("@bandwidth/webrtc");
```

Comment:
Definitely look at this file and how I did it. I'm not up to speed on JavaScript or Typescript conventions.

Reply:
So I guess you can't use `import` here? I think I ran into the same thing when I looked into it.

Reply:
Yeah, you can only use import when you have a type declarations file (.d.ts). It's a typescript thing. For packages without it you can use require plus the types/node package.